coverage/src/BamViewChart.js

```diff
@@ -485,12 +483,12 @@ function createBamView(bamHeader, data, element, bamViewControlsElement, broker)
         // Create a text label for showing the chromosome name and selected region and put it in the botton of the main chart
         svg.append('text')
             .attr('class', 'chromosome-label')
-            .attr('x', xScale(chromosomeEnd) / 2)
+            .attr('x', xScale(chromosomeLength) / 2)
             .attr('y', margin.top + mainHeight + 10)
             .attr('dy', '.35em')
             .attr('text-anchor', 'start')
             .attr('fill', 'black')
-            .text(`Chr ${orgChromosome}: 0 - ${chromosomeEnd} (${chromosomeEnd} bp)`)
+            .text(`${bamHeaderMap[chromosome].sn}:1-${chromosomeLength} (${chromosomeLength} bp)`)
```

Comment:
Can't you just us `chromosome` here instead of `bamHeaderMap[chromosome].sn`? I see this in other places as well

Reply:
No, we cannot use `chromosome` directly here. I tried before, then I found that the chromosome format in geneinfo is "chr1", but our bamHeader has two different format. E.g "chr1" or "1". So here we need to make sure it's always mapped to the only source of truth which is the bamHeaderMap.

Reply:
I'm confused. `bamHeaderMap` uses `bamHeader[x].sn` as its keys. Which is exactly what `chromosome` is in the line above. If the problem you just described exists, it should be happening with the current code as well as with my suggestion.

Reply:
Combined with my other suggestion, the final code should look like this:

```javascript
.text(`${bamHeader[indexMap[chromosome]].sn}:1-${chromosomeLength} (${chromosomeLength} bp)`)
```

If geneinfo needs a specific format, you can normalize for that based off the source of truth, which is `bamHeader`.

Reply:
> I'm confused. `bamHeaderMap` uses `bamHeader[x].sn` as its keys. Which is exactly what `chromosome` is in the line above. If the problem you just described exists, it should be happening with the current code as well as with my suggestion.

Sorry, I should give more details. In the function `brushToRegion` the pass in `chromosome` is from the geneinfo. That's why we cannot use the `chromosome` directly.

I changed the mapping logic based on your suggestion.